tools/clang/unittests/HLSL/CMakeLists.txt

```diff
@@ -104,8 +99,6 @@ if(WIN32)
 # Add includes for platform helpers and dxc API.
 include_directories(${TAEF_INCLUDE_DIRS})
 include_directories(${DIASDK_INCLUDE_DIRS})
-include_directories(${D3D12_INCLUDE_DIRS})
-include_directories(${LLVM_MAIN_INCLUDE_DIR}/dxc/Test)
```

Comment:
Lots of incidentals here. I noticed more differences than should exist between the platforms given the support on both currently. So I took them out.